app/src/main/res/values/strings.xml

```diff
@@ -64,6 +64,9 @@
   <string name="practice">Practice</string>
   <string name="revision">Revision</string>
   <string name="developer_options_activity_title">Developer Options</string>
+  <string name="mark_chapters_completed_activity_title">Mark Chapters Completed</string>
```

Comment:
though this file doesn't come under my codeowners, nit suggestion, It will be helpful to keep these new strings under a section just like we have a section for `  <!-- DeveloperOptionsFragment -->`

Reply:
I think we should create a separate issue to put all the activity titles under one section such as `<!-- Activity Titles -->`. WDYT?

Reply:
Sounds good to me, let's create a good first issue for any Home activity only. I will create a good first issue for other screens later on. 

For Developer Options, suggesting to fix in this PR only.

Reply:
I have arranged the titles. PTAL. Also, I have filed an issue for the same (#3384).

Reply:
Earlier I got the wrong idea from activity Titels, I think it will be replaced with the activity name. 

Suggest in review, take a look at comments below.

Reply:
Apologies for the delay. I have addressed all your below comments. PTAL. Thanks.